erpnext/selling/doctype/customer/customer.py

```diff
@@ -416,9 +418,12 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
 def send_emails(args):
 	args = json.loads(args)
 	subject = (_("Credit limit reached for customer {0}").format(args.get('customer')))
-	message = (_("Credit limit has been crossed for customer {0} ({1}/{2})")
-			.format(args.get('customer'), args.get('customer_outstanding'), args.get('credit_limit')))
-	frappe.sendmail(recipients=[args.get('credit_controller_users_list')], subject=subject, message=message)
+	message = (_("Credit limit has been crossed for customer {0} ({1}/{2})<br>Document Link: {3}").format(
+		args.get('customer'), args.get('customer_outstanding'), args.get('credit_limit'), args.get('document_link')))
+	frappe.sendmail(recipients=args.get('credit_controller_users_list'), subject=subject, message=message)
```

Comment:
- recipients are incorrectly sent , the name is attached along with the email, due to which emails arent going
- Also, Separate the html and concatenate to the string. HTML is not translatable, hence can't include it in _ method. Do this for the this whole method